.circleci/config.yml

```diff
@@ -0,0 +1,192 @@
+# steps, including restoring of cache, saving of cache.
+#
+# we also install `make` here.
+#
+# Adapted from https://github.com/lambdaisland/meta/blob/master/circleci/clojure_orb.yml
+
+commands:
+  with_cache:
+    description: |
+      Run a set of steps with Maven dependencies and Clojure classpath cache
+      files cached.
+      This command restores ~/.m2 and .cpcache if they were previously cached,
+      then runs the provided steps, and finally saves the cache.
+      The cache-key is generated based on the contents of `deps.edn` present in
+      the `working_directory`.
+    parameters:
+      steps:
+        type: steps
+      files:
+        description: Files to consider when creating the cache key
+        type: string
+        default: "deps.edn project.clj build.boot"
+      cache_version:
+        type: string
+        description: "Change this value to force a cache update"
+        default: "1"
+    steps:
+      - run:
+          name: Install make
+          command: |
+            sudo apt-get install make
+      - run:
+          name: Generate Cache Checksum
+          command: |
+            for file in << parameters.files >>
+            do
+              find . -name $file -exec cat {} +
+            done | shasum | awk '{print $1}' > /tmp/clojure_cache_seed
+      - restore_cache:
+          key: clojure-<< parameters.cache_version >>-{{ checksum "/tmp/clojure_cache_seed" }}
+      - steps: << parameters.steps >>
+      - save_cache:
+          paths:
+            - ~/.m2
+            - .cpcache
+            - repo
+          key: clojure-<< parameters.cache_version >>-{{ checksum "/tmp/clojure_cache_seed" }}
+
+# The jobs are relatively simple. One runs utility commands against
+# latest stable JDK + Clojure, the other against specified versions
+
+jobs:
+
+  util_job:
+    description: |
+      Running utility commands/checks (linter etc.)
+      Always uses Java11 and Clojure 1.10
+    parameters:
+      steps:
+        type: steps
+    executor: openjdk11
+    environment:
+      VERSION: "1.10"
+    steps:
+      - checkout
+      - with_cache:
+          cache_version: "1.10"
+          steps: << parameters.steps >>
+
+
+  test_code:
+    description: |
+      Run tests against given version of JDK and Clojure
+    parameters:
+      jdk_version:
+        description: Version of JDK to test against
+        type: string
+      clojure_version:
+        description: Version of Clojure to test against
+        type: string
+    executor: << parameters.jdk_version >>
+    environment:
+      VERSION: << parameters.clojure_version >>
+    steps:
+      - checkout
+      - with_cache:
+          cache_version: << parameters.clojure_version >>
+          steps:
+            - run:
+                name: Downloading Source deps
+                command: make source-deps
+            - run:
+                name: Running tests
+                command: make test
+            # - run:
```

Comment:
You can remove this, as I see it's in the Makefile already.

Reply:
Removed